plugins/modules/ec2_vpc_nat_gateway.py

```diff
@@ -544,13 +552,14 @@ def release_address(client, module, allocation_id):
 
 
 def create(client, module, subnet_id, allocation_id, tags, client_token=None,
```

Comment:
I could be nice to cover the `params` dict creation with a unit-tests.

Reply:
I would like to create a separate PR to add unit test cases for the complete module.